rosdep/python.yaml

```
@@ -6863,6 +6863,9 @@ python3-sphinx-rtd-theme:
 python3-sqlalchemy:
   debian: [python3-sqlalchemy]
   ubuntu: [python3-sqlalchemy]
+python3-sympy:
+  debian: [python3-sympy]
+  ubuntu: [python3-sympy]
```

```suggestion
  debian: [python3-sympy]
  fedora: [python3-sympy]
  gentoo: [dev-python/sympy]
  ubuntu: [python3-sympy]
```

* https://packages.gentoo.org/packages/dev-python/sympy
* http://www.rpmfind.info/linux/rpm2html/search.php?query=python3-sympy